proto/raw-js-openapi/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM deephaven/proto-backplane-grpc:local-build AS proto-backplane-grpc
+FROM --platform=linux/amd64 deephaven/proto-backplane-grpc:local-build AS proto-backplane-grpc
```

Comment:
cc @niloc132 . I debugged w/ Ryan a while ago, and we needed to make this change. I'm guessing some change in default Docker behavior.

Reply:
After a discussion with @niloc132 , I have reverted this change and cherry-picked a better change from his fork.